### ISSUE TITLE
provide example usage in neural network model

### ISSUE THREAD

**Issue by dwiel** [dwiel/tensorflow_hmm#4]

**Comment by dwiel** [comment]
I've got some code that I've used to test this hmm as a layer in a neural network, but it isn't ready for release yet and not documented.

**Comment by zenghsh3** [comment]
Hi, dwiel! Are you ready to release these code? I recently want to use tensorflow to train a model combined by cnn and hmm. I think these code will help me a lot! If you can help me I will be very grateful to you. Can you give me some advice?

**Comment by dwiel** [comment]
I actually haven't had time to clean these examples up, but I just committed some jupyter notebooks which may be of use.  You can check out this notebook in particular where I am experimenting with taking the gradient of the hmm in situations similar to using the hmm as the final layer of a neural network.  Let me know if you have any specific questions.

**Comment by zenghsh3** [comment]
OK, thanks for your help.

**Comment by Satishpas2** [comment]
Can I get the code where the hmm is added with neural network.
Thanks in advance

**Comment by dwiel** [comment]
Just went through the notebooks and updated them.  Started on a new notebook over the weekend that demonstrates use in neural network, but not quite done yet.

**Comment by zach-nervana** [comment]
I finally have an example Notebook which uses this as a layer in a neural network: https://github.com/dwiel/tensorflow_hmm/blob/master/notebooks/keras_example.ipynb
